### PR TITLE
feat: workflow to auto create PRs to merge main

### PR DESCRIPTION
**Pull request by paulbalaji**
feat: workflow to auto create PRs to merge main

### CODE REVIEW

File: .github/workflows/create-merge-prs.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  create-merge-prs:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        name: [ezeth, injective, nautilus, nexus]
```
Comment (jmrossy) [inline .github/workflows/create-merge-prs.yaml]: not nautilus, it's stuck in 2023

**Comment by jmrossy** [comment]
Cool!

**Comment by paulbalaji** [comment]
> @paulbalaji Cool! I will warn you that merge conflicts are not uncommon when I do these updates so about half the time I think we'll still need to do this by hand.

no worries - just thought i'd take a few mins to play around and find out

**Comment by paulbalaji** [comment]
woohoo! 

https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/229
https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/228
https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/227

File: .github/workflows/create-merge-prs.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Merge Main to Other Branches
+
+on:
+  push:
+    branches:
+      - main
+  # just for testing
+  pull_request:
+    branches:
+      - main
```
Comment (paulbalaji) [inline .github/workflows/create-merge-prs.yaml]: remove before merge
```suggestion
```

File: .github/workflows/create-merge-prs.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Create and push merge branch
+        run: |
+          git checkout ${{ matrix.name }}
+          git merge origin/main
+          git checkout -b main-to-${{ matrix.name }}
+          echo "testing" > dummy_test_file.txt
+          git add dummy_test_file.txt
+          git commit -m "chore: merge main into ${{ matrix.name }}"
```
Comment (paulbalaji) [inline .github/workflows/create-merge-prs.yaml]: remove before merge
```suggestion
```

**Pull request merged**